server/repository/src/db_diesel/changelog/README.md

```diff
@@ -0,0 +1,42 @@
+# Changelog
+
+A number of operations rely on processing a `queue` of database changes. There include pushing records to central server during synchronisation, processing transfer records, processing messaging, aggregation, interoperability synchronisation, etc.. 
```

Comment:
```suggestion
A number of operations rely on processing a `queue` of database changes.
These operations include pushing records to central server during synchronisation, processing transfer records, processing messaging, aggregation, interoperability synchronisation, etc.. 
```

To make reviewing easier I always put one sentence in one line when writing the docs. It makes it also much easier to find the sentence when you fix a reviewer comment.